session.go

```diff
@@ -353,9 +358,18 @@ func (s *Session) stopKeepalive() {
 	defer s.keepaliveLock.Unlock()
 	if s.keepaliveTimer != nil {
 		s.keepaliveTimer.Stop()
+		s.keepaliveTimer = nil
 	}
 }
 
+func (s *Session) extendKeepalive() {
+	s.keepaliveLock.Lock()
+	if s.keepaliveTimer != nil && !s.keepaliveActive {
+		s.keepaliveTimer.Reset(s.config.KeepAliveInterval)
```

Comment:
"Reset should be invoked only on stopped or expired timers with drained channels." - do we know that's the case here?

Reply:
already discussed in the previous pr...

Reply:
That's just bad documentation (https://github.com/libp2p/go-yamux/pull/16#discussion_r392064661). Reset will actually _call_ stop internally. The only reason you need to do it is if you need to drain the channel.

However, we're not even using a normal timer here.